decaylanguage/dec/decparser.py

```diff
@@ -200,25 +195,25 @@ def addline(self, line):
         spl = line.split()
         try:
             bf = float(spl[0])
-        except:
-            raise Exception("Cannot parse decay line: %s" % line)
-        ##        mod_found = False
-        ##         for x in known_models:
-        ##             if x in line:
-        ##                 mod_found = True
-        ##         if not mod_found:
-        ##             print '\n', line
+        except Exception:
+            raise RuntimeError("Cannot parse decay line: %s" % line)
```

Comment:
Always should be a bit more specific than "Exception" when throwing. RuntimeError is a pretty good "I don't know I want to throw" default.

Reply:
Yep.